docs/index.html

```diff
@@ -53,7 +53,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Asks for help when they need it.</li>
         <li>Supports code in production, even if they did not write it.</li>
         <li>Can feel like an impostor at times.</li>
-
+        <li>Is excited to attend a conference even online.</li>
```

Comment:
So I don't know if this would be true of all 1x engineers. I know many who've not attended conferences or haven't been super excited about it when they've been required to despite being excellent 1x engineers because it's not a format that they can learn well with and/or they're introverted and don't really like hallway tracking.

Given that, perhaps this would be a better approach to convey the same idea but in a more inclusive manner?

```suggestion
        <li>Can be excited when new opportunities present themselves.</li>
```

Since "Is open to learning" already exists, this rephrasing covers the other element of "attend a conference even online" which is new opportunities that present themselves.

Reply:
I agree, my thought was that a 1x Engineer is always looking for some opportunity to learn something new in this kind of events (my case), but is true that is not always a fact. Maybe the main idea is that

```suggestion
        <li>Likes to have the encouragement of someone else.</li>
```
or  something similar.

```suggestion
        <li>Likes to have the encouragement of someone else.</li>
        <li>Can be excited when new opportunities present themselves.</li>
```
Also this looks like a good option.

Reply:
What about:

```suggestion
        <li>Appreciates the encouragement from their peers.</li>
```

Reply:
Sounds perfect! :fire: